File: aula03-managed-point-java/PontoApp.java
class PontoApp{

	public static void main(String [] args)
	{
        Ponto p = new Ponto(5, 7, 9);
		p.print();
        String msg = String.format("p._x = %d", p._x);
        System.out.println(msg);
	}

}

File: aula04-verifier/PontoApp.cs
using System; // <=> import java.lang;

class PontoApp{

	public static void Main(String [] args)
	{
        Ponto p = new Ponto(5, 7, 9);
		p.Print();
        String msg = String.Format("p._x = {0}", p._x);
        Console.WriteLine(msg);
	}

}